packages/react-components/tsconfig.types.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
having multiple tsconfig is generally a good pratice, one for building, one for testing, one for linting.

i copied the naming `tsconfig.types.json` from react-spectrum. its a good name since this tsconfig is only used to generate type. The tsconfig at the root is used for linting

Reply:
Unfortunately, the include and outDir options can’t be hoisted in the root config because they are resolved relatively to the config they’re in.